dtfs-central-api/api-tests/mocks/test-users/api-test-users.js

```diff
@@ -0,0 +1,235 @@
+    emails: [
+      'maker4@ukexportfinance.gov.uk',
+      'checker4@ukexportfinance.gov.uk',
+    ],
+  },
+  HSBC: {
+    id: '961',
+    name: 'HSBC',
+    emails: [
+      'maker1@ukexportfinance.gov.uk',
+      'maker2@ukexportfinance.gov.uk',
+    ],
+  },
+  LLOYDS: {
+    id: '964',
+    name: 'LLOYDS',
+  },
+  RBS: {
+    id: '1004',
+    name: 'RBS',
+  },
+  Santander: {
+    id: '953',
+    name: 'Santander',
+  },
+  UKEF: {
+    id: '9',
+    name: 'UKEF test bank (Delegated)',
+    emails: [
+      'maker1@ukexportfinance.gov.uk',
+      'checker1@ukexportfinance.gov.uk',
+    ],
+  },
+  any: {
+    id: '*',
+  },
+};
+
+const testUsers = [
+  { username: 'no-roles', password: 'P@ssword1234', roles: [] },
+  { username: 'an-editor', password: 'P@ssword1234', roles: ['editor'] },
+  {
+    username: 'HSBC-maker-1',
+    password: 'P@ssword1234',
+    firstname: 'Mister',
+    surname: 'One',
+    email: 'one@email.com',
+    timezone: 'Europe/London',
+    roles: ['maker'],
+    bank: banks.HSBC,
+  },
+  {
+    username: 'HSBC-maker-2',
+    password: 'P@ssword1234',
+    firstname: 'Mister',
+    surname: 'Two',
+    email: 'two@email.com',
+    timezone: 'Europe/London',
+    roles: ['maker'],
+    bank: banks.HSBC,
+  },
+  {
+    username: 'Barclays-maker-1',
+    password: 'P@ssword1234',
+    firstname: 'Mister',
+    surname: 'Three',
+    email: 'three@email.com',
+    timezone: 'Europe/London',
+    roles: ['maker'],
+    bank: banks.Barclays,
+  },
+  {
+    username: 'Barclays-maker-2',
+    password: 'P@ssword1234',
+    firstname: 'Mister',
+    surname: 'Four',
+    email: 'four@email.com',
+    timezone: 'Europe/London',
+    roles: ['maker'],
+    bank: banks.Barclays,
+  },
+  {
+    username: 'Barclays-checker-1',
+    password: 'P@ssword1234',
+    firstname: 'Mister',
+    surname: 'Five',
+    email: 'five@email.com',
+    timezone: 'Europe/London',
+    roles: ['checker'],
+    bank: banks.Barclays,
+  },
+  {
+    username: 'Ukef-maker-1',
+    password: 'P@ssword1234',
+    firstname: 'Mister',
+    surname: 'Six',
+    email: 'six@email.com',
+    timezone: 'Europe/London',
+    roles: ['maker'],
+    bank: banks.UKEF,
+  },
+  {
+    username: 'super-user',
+    password: 'P@ssword1234',
+    firstname: 'Mister',
+    surname: 'Seven',
+    email: 'seven@email.com',
+    timezone: 'Europe/London',
+    roles: ['maker'],
+    bank: banks.any,
+  },
+  {
+    username: 'data-admin',
+    password: 'P@ssword1234',
+    firstname: 'Mister',
+    surname: 'Eight',
+    email: 'eight@email.com',
+    timezone: 'Europe/London',
+    roles: ['data-admin'],
+  },
+  {
+    username: 'Barclays-maker-checker-1',
+    password: 'P@ssword1234',
+    firstname: 'Mister',
+    surname: 'Nine',
+    email: 'nine@email.com',
+    timezone: 'Europe/London',
+    roles: ['maker', 'checker'],
+    bank: banks.Barclays,
+  },
+
+  {
+    username: 'maker-tfm',
+    password: 'AbC!2345',
+    firstname: 'Tamil',
```

Comment:
do we need all of these users here? can we keep the list short if we are using just one/two?

Reply:
done